docs/content/faq/how-can-i-pass-configuration-to-my-app.md

```diff
@@ -0,0 +1,7 @@
+# How can I pass configuration to my app?
+
+The recommended way to pass configuration to your app is via environment variables, as is commonly done with Heroku or Docker.
+
+Flynn deploys apps directly from a git repository without a configuration management system such as Chef or Puppet to handle configuration files, and storing configuration -- in particular secrets -- in git is generally discouraged.
+
+This has the added benefits of supporting multiple arbitrary environments without a dedicated configuration file for each, and making configuration changes without going through the cycle of code change, commit, code review, and deploy.
```

Comment:
We should probably give an example of setting an environment variable, and advise users that doing so will start fresh jobs